gateway/src/apicast/metrics/3scale_backend_status.lua

```diff
@@ -7,7 +7,7 @@ local _M = {}
 
 local backend_response_metric = prometheus(
   'counter',
-  'backend_response',
+  'threescale_backend_response',
```

Comment:
Can they start with a number? `threescale` should be the last resort how to write`3scale`

Reply:
no, that's why I chose `threescale` :D

Reply:
That does not sound that bad. I'm fine with both.

Reply:
I'll add the `apicast` prefix. I think it makes sense if we take into account that other apps can export metrics to the same Prometheus instance. Also, it's recommended in the Prometheus guidelines: https://prometheus.io/docs/practices/naming/

Reply:
👍